packages/web-react/src/components/UNSTABLE_Header/README.md

```diff
@@ -0,0 +1,177 @@
+# UNSTABLE Header
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+The `UNSTABLE_Header` component is planned to replace the `Header` component in the future.
+
+This component provides these components:
+
+- [UNSTABLE_Header](#unstable-header)
+- [UNSTABLE_HeaderLogo](#unstable-headerlogo)
+
+## UNSTABLE Header
+
+The `UNSTABLE_Header` component is a main wrapper which provides mainly the visual for the Header.
+
+```jsx
+import { UNSTABLE_Header } from '@lmc-eu/spirit-web-react';
+
+<UNSTABLE_Header>{/* Content go here */}</UNSTABLE_Header>;
+```
+
+It also sets CSS variable for the Header height which can be used in other nested components.
+
+### API
+
+| Name       | Type                    | Default | Required | Description                    |
+| ---------- | ----------------------- | ------- | -------- | ------------------------------ |
+| `children` | `string` \| `ReactNode` | `null`  | ✓        | Content of the UNSTABLE_Header |
+
+The components accept [additional attributes][readme-additional-attributes].
+If you need more control over the styling of a component, you can use [style props][readme-style-props]
+and [escape hatches][readme-escape-hatches].
+
+## UNSTABLE HeaderLogo
```

Comment:
**suggestion (if-minor):** I think that this subcomponent does not need to be `UNSTABLE_`. It is new and unique and not in a collision like a Header.

Reply:
It is a subcomponent of UNSTABLE_Header. If it would be called just HeaderLogo I would look for it in Header, where it is not placed. Also it would be confusing in (S)CSS. 

Also we can benefit from the instability and bring BCs disguised as fixes/features if we find out it does not work :)